joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/AstCreatorBase.scala

```diff
@@ -98,13 +98,13 @@ abstract class AstCreatorBase(filename: String) {
   def controlStructureAst(
     controlStructureNode: NewControlStructure,
     condition: Option[Ast],
-    children: List[Ast] = List(),
+    children: Seq[Ast] = Seq(),
```

Comment:
Some frontends work with `List`s and others with `Seq`s. Thus it seemed natural to simply generalize than to convert to `List` in a few specific places.